packages/flutter/test/material/time_picker_test.dart

```diff
@@ -2162,6 +2163,55 @@ void main() {
 
     expect(tester.getSize(findBorderPainter().first), const Size(96.0, 70.0));
   });
+
+  testWidgets(
```

Comment:
Adding a link to the issue here will be helpful for future maintainers :)

```suggestion
  });

  // Regression test for https://github.com/flutter/flutter/issues/162229.
  testWidgets(
```